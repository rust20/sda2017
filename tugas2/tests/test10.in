15 6 5 5000 110 110
Arturia Pendragon;1;100;pedang
Amakusha;1;99;pedang
Sishou;1;49;panah
Atlanta;1;10;panah
Emiye;25;10;panah
Queen Riana;0;9999;pedang
Raven;45;12000;panah
Rigel;40;30000;panah
Stella;20;2000;pedang
Jyanu;22;100;pedang
Hector;1;11;panah
Hercules;20;10000;pedang
Gilgamesh;1;9999;panah
Cu;1;100;panah
Kizuna;20;1000;pedang
1;110;Gilgamesh,Hercules
3;2;Arturia Pendragon,Amakusha,Sishou
1;3;Atlanta,Emiye,Jyanu,Hector,Queen Riana
1;5;Rigel,Raven
109;110;Cu,Stella
107;1;Kizuna
1;107;11;1;panah;3
109;2;215;1;panah;3
2;1;40;13;pedang;4
110;1;201;300;pedang;1
110;110;10000;200;pedang;1000
..S#S.......................................#.............................................................D..S
D.##...........................................#..............................................................
#S##..........................................................................................................
#.#.................................#.........................................................................
#.#.......................................#...................................................................
#.#...........................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
......................................M.......................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
S.............................................................................................................
..............................................................................................................
.D...........................................................................................................S
D............................................................................................................D

