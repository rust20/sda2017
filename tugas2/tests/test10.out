1,107 RUN, kekuatan maksimal sejumlah: 0
1,110 Pahlawan yang ikut:Hercules,Gilgamesh
109,110 Pahlawan yang ikut:Stella,Cu
110,110 BATTLE, kekuatan: 32198, pahlawan: Gilgamesh,Hercules,Stella,Cu
1,5 Pahlawan yang ikut:Rigel,Raven
3,2 Pahlawan yang ikut:Arturia Pendragon,Amakusha,Sishou
1,3 Pahlawan yang ikut:Queen Riana,Jyanu,Hector,Atlanta,Emiye
2,1 BATTLE, kekuatan: 113998, pahlawan: Rigel,Raven,Gilgamesh,Hercules
109,2 BATTLE, kekuatan: 51999, pahlawan: Rigel,Raven,Gilgamesh
110,1 BATTLE, kekuatan: 60000, pahlawan: Rigel
107,1 Pahlawan yang ikut:Kizuna
Akhir petualangan Gudako
Level Gudako: 1156
Level pahlawan:
Rigel: 315
Gilgamesh: 215
Hercules: 214
Stella: 201
Cu: 201
Raven: 15
Arturia Pendragon: 1
Amakusha: 1
Sishou: 1
Queen Riana: 1
Jyanu: 1
Hector: 1
Atlanta: 1
Emiye: 1
Kizuna: 1
